portal-web/test/functional/com/liferay/portalweb/paths/portlet/forms/FormFields.path

```diff
@@ -359,6 +359,16 @@
 	<td>//a[contains(@title,"${key_titleName}")]</td>
 	<td></td>
 </tr>
+<tr>
+	<td>ASSERT_PARAGRAPH_TEXT_STYLE</td>
+	<td>//div[(@class='liferay-ddm-form-field-paragraph-text')]//${key_formattingStyle}[contains(text(),'Test')]</td>
+	<td></td>
+</tr>
+<tr>
+	<td>IFRAME_FORMATTING_STYLES</td>
```

Comment:
Hey @ewertonluna , I've changed the title of this path in order to follow the standard of other paths on this file.
Please see: https://github.com/emanuelfisilva/liferay-portal/pull/167/commits/0997789e35117592a49fe7a4574ed0a06e9227ba